_posts/example-2.md

```diff
@@ -8,6 +8,7 @@ author:
   avatar: '/assets/authors/hoang.jpeg'
 ogImage:
   url: '/assets/posts/example/cover.png'
+tags: 'tag1, tag2'
```

Comment:
Could we use the array like this?
```suggestion
tags: ['tag1', 'tag2']
```

That is YML array: https://www.w3schools.io/file/yaml-arrays/

Reply:
sure, done in https://github.com/nimblehq/til-web/pull/25/commits/6f1eca582bad6457d1ad4ab1d18c6bc36bad3014, Thanks for providing the link!